tests/c_languages/CPPCheckBearTest.py

```diff
@@ -1,37 +1,50 @@
+import os
+import unittest
+
+from queue import Queue
+
 from bears.c_languages.CPPCheckBear import CPPCheckBear
-from coalib.testing.LocalBearTestHelper import verify_local_bear
-
-good_file = """
-using namespace std;
-int main() {
-    cout << "Hello, world!" << endl;
-    return 0;
-}"""
-
-warn_file = """
-void f1(struct fred_t *p)
-{
-    int x;
-    if (p)
-        do_something(x);
-}"""
-
-bad_file = """
-#define f(c) { \
-    char s[10]; \
-    s[c] = 42; \
-}
-int main() {
-    f(100);
-    return 0;
-}"""
-
-
-CPPCheckBearTest1 = verify_local_bear(CPPCheckBear,
-                                      valid_files=(good_file, warn_file),
-                                      invalid_files=(bad_file,))
-
-CPPCheckBearTest2 = verify_local_bear(CPPCheckBear,
-                                      valid_files=(good_file,),
-                                      invalid_files=(warn_file, bad_file),
-                                      settings={'enable': 'unusedFunction'})
```

Comment:
If you are converting tests to your "new" style, I would request another commit prepended to this one that does the refactor. Makes review easier^^

Reply:
@Makman2 I have made the required changes.